### ISSUE TITLE
API: Swap the testing db with SQLite

### ISSUE THREAD

**Issue by rajdip-b** [keyshade-xyz/keyshade#333]
## Description

We want to swap out our Postgres database in docker compose file to SQLite. This will improve the E2E execution speed
https://github.com/keyshade-xyz/keyshade/blob/b29ac91c5216fc82fd101f2deafba4af9835bb4d/docker-compose-test.yml#L3-L14

## Solution

- Remove the db service in `docker-compose-test.yml`
- Update `schema.prisma` to be compatible with [Prisma SQLite](https://www.prisma.io/docs/orm/overview/databases/sqlite)

**Comment by rajdip-b** [comment]
Hey @MelloB1989! Unfortunately I can't assign this issue to you since @vr-varad has already been assigned to this. Only if @vr-varad thinks that he wants to drop his attempt, can I assign you.

**Comment by vr-varad** [comment]
Sorry I haven't made a pr but has worked on it. @rajdip-b thought would make in fosshack weekend.
Should I make a pr?

**Comment by rajdip-b** [comment]
No, hold on to that. Otherwise it won't be considered for the hackathon.
